test/e2e/certificates_test.go

```diff
@@ -267,6 +270,70 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate in GCP mint/passthrough clusters-65035", func() {
```

Comment:
```suggestion
		It("should obtain a valid LetsEncrypt certificate in GCP mint/passthrough cluster", func() {
```
In the public repo, I think it's better to not add our private case id. In future, we would find a better way to tracking/mapping cases.